From 0ffd51e43cdc9d294a6765f025b3afa537ff632d Mon Sep 17 00:00:00 2001
From: Ben Finney <ben+debian@benfinney.id.au>
Date: Wed, 5 Apr 2017 19:24:54 +1000
Subject: [PATCH 2/5] =?UTF-8?q?Correct=20=E2=80=9Cpreferred=20form=20of=20?=
 =?UTF-8?q?the=20work=20for=20making=20changes=20to=20it=E2=80=9D=20in=20m?=
 =?UTF-8?q?anpage.?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The phrase “preferred form of modification” does not refer to a form
of the work. It also does not match the GNU GPL's definition of
“source”.

Instead, the GNU GPL (version 3 §1) defines “source code” as the
“preferred form of the work for making modifications to it”.
---
 doc/suspicious-source.1 | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/doc/suspicious-source.1 b/doc/suspicious-source.1
index 0cd5b188..045a0c64 100644
--- a/doc/suspicious-source.1
+++ b/doc/suspicious-source.1
@@ -15,18 +15,18 @@
 .TH SUSPICIOUS\-SOURCE 1 "Debian Utilities" "DEBIAN"
 
 .SH NAME
-suspicious\-source \- search for files that are not the GPL's
-"preferred form of modification"
+suspicious\-source \- search for files that do not meet the GPL's
+definition of "source" for a work
 
 .SH SYNOPSIS
 \fBsuspicious\-source\fP [\fIoptions\fR]
 
 .SH DESCRIPTION
-\fBsuspicious\-source\fP is a script that outputs a list of files which
-are not common source files.
-This should be run in the root of a source tree to find files which might
-not be the "preferred form of modification" that the GPL and other licenses
-require.
+\fBsuspicious\-source\fP outputs a list of files which are probably not
+the source form of a work.
+This should be run in the root of a source tree to find files which
+might not be, in the definition from the GNU GPL, the "preferred form
+of the work for making modifications to it".
 .PP
 The files inside version control system directories (like
 \fI.bzr/\fR or \fICVS/\fR) are not considered.
-- 
2.11.0

